docs/client/full-api/api/pubsub.md

```diff
@@ -173,10 +173,11 @@ collection name.
 The client will see a document if the document is currently in the published
 record set of any of its subscriptions.
 
-The `onReady` callback is called with no arguments when the server
-[marks the subscription as ready](#publish_ready). The `onError` callback is
-called with a [`Meteor.Error`](#meteor_error) if the subscription fails or is
-terminated by the server.
+The `onReady` callback is called with no arguments when the server [marks the
+subscription as ready](#publish_ready). The `onStopped` callback is called with
+a [`Meteor.Error`](#meteor_error) if the subscription fails or is terminated by
+the server. If the subscription is stopped for any other reason, `onStopped` is
```

Comment:
I don't think we have a history of documenting deprecated things. There's a general issue here where it's really hard to access the docs of old meteor versions. Maybe it's worth uploading all of the old docs apps with URLs like "docs-1.0.3.1.meteor.com" or something